### ISSUE TITLE
Similar to other issues, enemies no longer melee attack

### ISSUE THREAD

**Issue by WylerL** [ShikyoKira/Project-New-Reign---Nemesis-Main#255]
So after installing a few mods with Nemesis (Combat Gameplay Overhaul, Archery Gameplay Overhaul, 360 Movement Behavior and Sexlab Framework) human enemies are broken. If they have a melee weapon of seemingly any form they don't use it, instead just standing around and repositioning themselves in a fight. They will bash with their weapon or shield if they have one. If an enemy near by them is on low enough health, they can also complete killmoves. It is just their melee attack that seems broken. And my own animations and mod functions work great, just the enemies seem to have forgotten how to hit things, it is very frustrating.

**Comment by WylerL** [comment]
One thing to also add, if they have a bow, crossbow or spells to attack with and ammo/magicka, they will attack with them normally.

**Comment by WylerL** [comment]
Also if it helps, I am playing Oldrim with the correct version of Nemesis. I also use MO2 as my mod manager. And at this point, from what I understand I have done everything correctly. This is the only feature of any of the 100+ mods I am running I find to not be working, but obviously it is a bit of a big issue.

**Comment by WylerL** [comment]
At this point I can confirm after running without any Nemesis related files active, and then enabling all of them again one at a time/isolated and even together, Nemesis is causing this issue. It does not matter what mods I run through Nemesis, just after running the engine with its files active I come back to the game and no one can attack. Going to swap back to FNIS for the few mods I can salvage and continue on with my playthrough. Hopefully this issue can be helped as I love Nemesis and wish I could use it.

**Comment by ShikyoKira** [comment]
Duplicate of #180 
Looking into it. Thanks for the rigorously testing

**Comment by Wiubiki** [comment]
I had a similar issue with melee attack animation from non-humanoid hostiles while trying out both Nemesis and later Pandora. I managed to solve by running the game stock and adding mods one by one. Check [this](https://github.com/Monitor221hz/Pandora-Behaviour-Engine-Plus/discussions/579#discussion-9237053) out for more details and in case it helps out. 
tl;dr: it was caused by aTweaks & Utilities
